### PR TITLE
output help information when dapr has no parameters

### PR DESCRIPTION
**Pull request by tianyax**
# Description

Fixed the problem that the output of the dapr did not match the documentation after installation.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1254

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

**Review by shubham1172 — CHANGES_REQUESTED** [review]
The `--version` flag should be the same as `dapr version` IMO. It is meaningful to provide both CLI and installed runtime's version.

File: cmd/dapr.go [hunk 1]
```diff
@@ -39,11 +39,7 @@ var RootCmd = &cobra.Command{
 
 ===============================
 Distributed Application Runtime`,
-	Run: func(cmd *cobra.Command, args []string) {
-		if versionFlag {
-			printVersion()
-		}
```
Comment (mukundansundar) [inline cmd/dapr.go:45]: I think adding return inside the block and having command print usage outside the block should solve this issue.

**Comment by tianyax** [comment]
> The `--version` flag should be the same as `dapr version` IMO. It is meaningful to provide both CLI and installed runtime's version.

@shubham1172  For `--version` flag I think in most cases it should print its own information, and it overlaps with the `dapr version` function, I respect the original design and will make a change.😊

File: cmd/dapr.go (hunk 1 above)
Comment (tianyax) [reply cmd/dapr.go:45]: 👍，Thanks

**Review by mukundansundar — APPROVED** [review]
LGTM

**Comment by mukundansundar** [comment]
@tianyax Thanks for the contribution, welcome to the Dapr community.

**Pull request merged**